moto/ec2/models.py

```diff
@@ -6421,7 +6414,7 @@ def __init__(
         self.port_range_to = port_range_to
 
 
-class VpnGateway(TaggedEC2Resource):
+class VpnGateway(CloudFormationModel, TaggedEC2Resource):
```

Comment:
This was more of a side-effect when fixing this issue, as I had to be able to create a VPNGateway first in order to attach it to a VPN